.github/lock.yml

```
@@ -0,0 +1,37 @@
+# Configuration for Lock Threads - https://github.com/dessant/lock-threads
+
+# Number of days of inactivity before a closed issue or pull request is locked
+daysUntilLock: 60
```

I would set this for longer -- the problem cases have been when someone comments on an issue from e.g. January 2018 when it's June 2019.

```suggestion
daysUntilLock: 180
```